contrib/download-frozen-image-v2.sh

```diff
@@ -63,10 +63,10 @@ fetch_blob() {
 			-o "$targetFile" \
 			-D-
 	)"
-	curlHeaders="$(echo "$curlHeaders" | tr -d '\r')"
-	if [ "$(echo "$curlHeaders" | awk 'NR == 1 { print $2; exit }')" != '200' ]; then
+	if echo "$curlHeaders" | egrep -q "^HTTP/[0-9].[0-9] 307"; then
```

Comment:
Also, would it make sense to allow for any redirection code here (instead of just `307`)?  ala https://en.wikipedia.org/wiki/URL_redirection#HTTP_status_codes_3xx